vm/qemu/qemu.go

```diff
@@ -646,6 +645,9 @@ func (inst *instance) Diagnose(rep *report.Report) ([]byte, bool) {
 	// TODO: we don't need registers on all reports. Probably only relevant for "crashes"
 	// (NULL derefs, paging faults, etc), but is not useful for WARNING/BUG/HANG (?).
 	ret := []byte(fmt.Sprintf("%s Registers:\n", time.Now().Format("15:04:05 ")))
+	if _, err := inst.qmp(&qmpCommand{Execute: "stop"}); err != nil {
```

Comment:
I suspect this may truncate delayed kernel output.
vm.extractError calls Diagnose immediately after noticing first oops line, then we wait for 10 seconds to give kernel print whole oops messages. If Diagnose stops the VM, kernel may not be able to finish printing the report.
I am also not sure this is useful. We issue stop milliseconds after the oops, registers can be already arbitrary off.